src/RoutingControllersOptions.ts

```diff
@@ -85,4 +86,28 @@ export interface RoutingControllersOptions {
      */
     currentUserChecker?: CurrentUserChecker;
     
+    /**
+     * Default settings
+     */
+    defaults?: {
+        /**
+         * If set, all null responses will return specified status code by default
+         */
+        nullResultCode?: number;
+
+        /**
+         * If set, all undefined responses will return specified status code by default
+         */
+        undefinedResultCode?: number;
+
+        /**
+         * Default param options
+         */
+        paramOptions?: {
```

Comment:
Okay lets left it as it is if you both think its better. But Im still against such redundant doubtful-value nestings which only brings complexity.